app/controllers/bigbluebutton_api_controller.rb

```diff
@@ -380,7 +380,12 @@ def recordings_disabled
   # Filter out unneeded params when passing through to join and create calls
   # Has to be to_unsafe_hash since to_h only accepts permitted attributes
   def pass_through_params
-    params.except(:format, :controller, :action, :checksum).to_unsafe_hash
+    # Get list of params that should not be modified by create/join API calls
+    excluded_params = Rails.configuration.x.send("#{action_name}_exclude_params")
```

Comment:
Rather than look up the configuration from the action name, please have the controller action pass the correct excluded parameters list as an argument to the `pass_through_params` method.